### PR TITLE
Added batch scripts to copy localizable resources

### PR DESCRIPTION
**Pull request by Benglin**
Background
-----
This pull request introduces `ExtractResources.bat` to copy only localizable resource files to a predefined `bin\AnyCPU\Debug\wwl` or `bin\AnyCPU\Release\wwl` folders (depending on the build flavor).

This is the structure of resulting `wwl` folder:

    wwl\en-US                           (*.resources.dll, *.xml)
    wwl\libg_locale\en_US\LC_MESSAGES   (*.mo, *.po)
    wwl\nodes\en-US                     (*.resources.dll)
    wwl\samples\en-US                   (sample dyn files)

    wwl\Revit_2014\en-US                (*.resources.dll, *.xml)
    wwl\Revit_2014\nodes\en-US          (*.resources.dll)

    wwl\Revit_2015\en-US                (*.resources.dll, *.xml)
    wwl\Revit_2015\nodes\en-US          (*.resources.dll)

    wwl\Revit_2016\en-US                (*.resources.dll, *.xml)
    wwl\Revit_2016\nodes\en-US          (*.resources.dll)

Also note that both `README.txt` and `License.rtf` files are moved out of `en-US` folder into `bin` folder, as they are no longer localizable.

Hi @nguyen-binh-minh, please take a look. Thanks!

### CODE REVIEW

**Comment by nguyen-binh-minh** [comment]
@Benglin LGTM

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -324,28 +324,17 @@ limitations under the License.
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
```
Comment (Benglin) [inline src/DynamoCore/DynamoCore.csproj]: Be gone, ancient stuff.

File: tools/install/DynamoInstaller.iss [hunk 2]
```diff
@@ -70,7 +70,6 @@ Source: temp\bin\*; DestDir: {app}; Flags: ignoreversion overwritereadonly; Comp
 Source: temp\bin\nodes\*; DestDir: {app}\nodes; Flags: ignoreversion overwritereadonly recursesubdirs; Components: DynamoCore
 Source: temp\bin\lang\*; DestDir: {app}\; Flags:skipifsourcedoesntexist ignoreversion overwritereadonly recursesubdirs; Components: DynamoCore
 Source: Extra\IronPython-2.7.3.msi; DestDir: {tmp}; Flags: deleteafterinstall;
-Source: temp\bin\lang\{#locale}\README.txt; DestDir: {app}\{#locale}\; Flags: onlyifdoesntexist isreadme ignoreversion; Components: DynamoCore
```
Comment (nguyen-binh-minh) [inline tools/install/DynamoInstaller.iss]: @Benglin if you remove this line the "View Readme" check box won't show up when installation finishes
May be just
Source: temp\bin\README.txt; DestDir: {app}\; Flags: onlyifdoesntexist isreadme ignoreversion; Components: DynamoCore

Comment (Benglin) [inline tools/install/DynamoInstaller.iss]: Good catch! Let me fix it.

**Comment by nguyen-binh-minh** [comment]
@Benglin LGTM

**Comment by Benglin** [comment]
Thanks @nguyen-binh-minh!

**Pull request merged**